test/integration/integration.test.ts

```diff
@@ -1135,3 +1136,66 @@ test('calls eslintrc function with repository and its location', async () => {
         "
     `);
 });
+
+test('loads typescript configuration file', async () => {
+    const { output } = await runProductionBuild(
+        {
+            CI: true,
+            rulesUnderTesting: [],
+            eslintrc: { root: true },
+            onComplete: `function onComplete() {
```

Comment:
I didn't realise that this'd work - in that case, we should remove the `// @ts-nocheck` [here](https://github.com/AriPerkkio/eslint-remote-tester/pull/320/files#diff-1a258b987746216dc81c746069ca5944c0f1ec848aa095a8a9f109a976dc2343R64) as we should be able to properly type this function now.